backend/person-service/src/main/kotlin/com/nosql/personservice/controller/PersonController.kt

```diff
@@ -0,0 +1,43 @@
+package com.nosql.personservice.controller
+
+import com.nosql.personservice.constants.url.PUBLIC_API_V1_PERSON_URL_PATH
+import com.nosql.personservice.dto.PersonDto
+import com.nosql.personservice.service.PersonService
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import javax.validation.Valid
+
+@RestController
+@RequestMapping(PUBLIC_API_V1_PERSON_URL_PATH)
+class PersonController(
+    private val personService: PersonService,
+) {
+
+    @PostMapping(
+        path = [SIGN_UP_URL_PATH],
+        consumes = [MediaType.APPLICATION_JSON_VALUE],
+        produces = [MediaType.APPLICATION_JSON_VALUE],
+    )
+    suspend fun signUp(
```

Comment:
signUp?

Reply:
Yes, it is request to add person info while registration